### DISCUSSION TITLE
Add LMDH to Grade A Endurance

### DISCUSSION THREAD

**Discussion by jam94mac** [abesto/ams2-career#76] [Ideas]
Loving the career mode! Is it possible to add the LMDH cars as a class option at the top of endurance?

**Comment by abesto** [comment]
Always happy to hear people enjoy the app!

https://github.com/abesto/ams2-career/issues/67#issuecomment-1374121325 outlines what it'd take to update the "official" car / track list.

Basically, someone (who is not me) would need to fill in the "car class" x "track layout" matrix. If that person steps up, we can get the app updated with all the new content!

**Reply by jam94mac** [reply]
Let me take a look and see what I can do.

**Reply by jam94mac** [reply]
thanks for the response btw!

**Reply by abesto** [reply]
I took care of a bunch of tech debt and caught up with two years of JS ecosystem changes today in the `dev` branch, so there should be no technical blockers to building & releasing a new version. LMK if you want to actually dive into this.
